.github/workflows/ci.yml

```diff
@@ -357,4 +357,4 @@ jobs:
           token: ${{ secrets.GITHUB_TOKEN }}
           resultPath: lib/coverage_results/.last_run.json
           failedThreshold: 93.5
-          failedThresholdBranch: 71.5
```

Comment:
Real coverage is quite high. The fractured, concurrent testing performed by GHA necessitates this lower number for now until we come up with a solution.